content/id/docs/concepts/workloads/controllers/garbage-collection.md

```diff
@@ -0,0 +1,133 @@
+```
+
+Keluaran menunjukkan bahwa pemilik Pod adalah sebuah ReplicaSet bernama `my-repset`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  ...
+  ownerReferences:
+  - apiVersion: apps/v1
+    controller: true
+    blockOwnerDeletion: true
+    kind: ReplicaSet
+    name: my-repset
+    uid: d9607e19-f88f-11e6-a518-42010a800195
+  ...
+```
+{{< note >}}
+Referensi pemilik lintas _namespace_ tidak diperbolehkan oleh desain. Artinya:
+1) _Namespace-scoped dependent_ hanya bisa menspesifikasikan pemilik jika berada di _namespace_ yang sama, dan pemilik bersifat _cluster-scoped_. 
+2) _Cluster-scoped dependent_ hanya bisa menspesifikasikan pemilik bersifat _cluster-scoped_, tetapi tidak berlaku untuk pemilik bersifat _namespace-scoped_.
+{{< /note >}}
+
+## Mengontrol bagaimana _garbage collector_ menghapus _dependent_
+
+Ketika kamu menghapus sebuah objek, kamu bisa menspesifikasi apakah _dependent_ objek tersebut juga dihapus secara otomatis. Menghapus _dependent_ secara otomatis disebut _cascading deletion_. _Cascading deletion_ memiliki dua mode: _background_ dan _foreground_
+
+### Foreground cascading deletion
+
+Pada *foreground cascading deletion*, pertama objek utama akan memasuki keadaan "_deletion in progress_". Pada saat keadaan "_deletion in progress_", kondisi-kondisi berikut bernilai benar:
+
+ * Objek masih terlihat via REST API
+ * `deletionTimestamp` objek telah ditentukan
+ * `metadata.finalizers` objek memiliki nilai `foregroundDeletion`.
+
+ Ketika dalam keadaan "_deletion in progress_", _garbage collector_ menghapus _dependent_ dari objek. Ketika _garbage collector_ telah menghapus semua "_blocking_" _dependent_ (objek dengan `ownerReference.blockOwnerDeleteion=true`), _garbage collector_ menghapus objek pemilik.
+
+ Jika kolom `ownerReferences` sebuah objek ditentukan oleh sebuah _controller_ (seperti Deployment atau Replicaset), `blockOwnerDeletion` akan ditentukan secara otomatis dan kamu tidak perlu memodifikasi kolom ini secara manual.
+
+ ### Background cascading deletion
+ 
+ Pada *background cascading deletion*, Kubernetes segera menghapus objek pemilik dan _garbage collector_ kemudian menghapus _dependent_ pada _background_.
+
+ ### Mengatur kebijakan _cascading deletion_
+
+ Untuk mengatur kebijakan _cascading deletion_, tambahkan kolom `propagationPolicy` pada argumen `deleteOptions` ketika menghapus sebuah Object. Nilai yang dapat digunakan adalah "Orphan", "Foreground", atau "Background".
+
+ Sebelum Kubernetes 1.9, kebijakan default _garbage collection_ untuk banyak _resource controller_ adalah **orphan**. Ini meliputi ReplicationController, ReplicaSet, StatefulSet, DaemonSet, dan Deployment. Untuk jenis pada kelompok versi `extensions/v1beta1`, `apps/v1beta1`, dan `apps/v1beta2`, kecuali kamu menspesifikasikan dengan cara lain, objek _dependent_ adalah _orphan_ secara default. Pada Kubernetes 1.9, untuk semua jenis pada kelompok versi `apps/v1`, objek _dependent_ dihapus secara default.
```

Comment:
make default italicized

Reply:
fixed in https://github.com/kubernetes/website/pull/14513/commits/3842f1701da6c7406f2771419b0192e6ca5f7305